parm/archive/gfswave.yaml.j2

```diff
@@ -0,0 +1,19 @@
+    gfswave:
+        {% set head = "gfswave.t" + cycle_HH + "z." %}
+        name: "GFSWAVE"
+        target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gfswave.tar"
+        mandatory:
+            {% for fh in range(0, FHMAX_HF_WAV + FHOUT_HF_WAV, FHOUT_HF_WAV) %}
+            # NOTE This is as explicit as possible without major logic to parse wavepostGRD.
+            #      Matches files of the form "gfswave.tCCz.<region>.<res>.fHHH.grib2".
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2"
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2.idx"
+            {% endfor %}
+            {% for fh in range(FHMAX_HF_WAV + FHOUT_WAV, FHMAX_WAV_GFS + FHOUT_WAV, FHOUT_WAV) %}
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2"
+            - "{{ COM_WAVE_GRID | relpath(ROTDIR) }}/{{ head }}*.*.f{{ '%03d' % fh }}.grib2.idx"
+            {% endfor %}
+        optional:
```

Comment:
Apologies, I haven't read through the full PR yet so this might be obvious from other parts of the code, but what is the difference between mandatory and optional?

Reply:
Mandatory files will raise a fatal error if they are not found during archiving.  Optional files will print warnings to the log file.

Reply:
As long as we have point output, I would think these should be mandatory too...

Reply:
Speaking to that, I think I "required" makes more sense than "mandatory", so I will update that.

Reply:
so {{ head }} in this case is just gfswave?  We do have a few other tar files that are only occationally created if DOIBP_WAV=YES, but we'd want those archived if they are created.

Reply:
Wave point data files have been made required.  The change was tested and works for a forecast-only S2SW case.

Reply:
Yes, that's what it is here.  If there's a case where additional data needs to be included in this file that does not start with `gfswave`, we can find a solution to add it.  `head` isn't a hard-fast prefix for all files in a given template.